src/SFA.DAS.Tools.Servicebus.Support.Web/Views/MessageList/Index.cshtml

```diff
@@ -154,11 +119,36 @@
     <div>
         <fieldset>
             <legend>Message actions</legend>
-            <a class="govuk-link" asp-controller="MessageList" asp-action="ReceiveMessages" asp-route-queue="@Model.QueueInfo.Name" onclick="displayBusyIndicator()" title="Get any further messages that may be on the queue">Get more</a> |
-            <a class="govuk-link" id="btnReplayMessages" href="#" title="Move selected messages back to the processing queue">Move selected to processing queue</a> |
+
+            <select name="getQty" id="getQty">
+                <option value="250">250</option>
+                <option value="500">500</option>
+                <option value="1000">1000</option>
+            </select>
+            <a class="govuk-link" id="btnReceiveMessages" href="#" onclick='getQueueMessages("@Model.QueueInfo.Name")' title="Get any further messages that may be on the queue">Get more</a> |
```

Comment:
You could change the onclick to be `getQueueMessages()` and put the `@Model.QueueInfo.Name` as a javascript variable in the function ready to go to make this link easier to digest?

Reply:
done